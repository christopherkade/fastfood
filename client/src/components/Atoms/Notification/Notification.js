import React from "react"
import PropTypes from 'prop-types';
import styled from "styled-components"

const StyledNotification = styled.div`
  position: fixed;
  bottom: 1rem;
  right: 1rem;
  background-color: #D52941;
  padding: 1rem;
  border-radius: 4px;
  color: white;
`

const Notification = ({ error }) => {
  return (
    <>
      {
        error ?
          <StyledNotification>
            {error}
          </StyledNotification > : null
      }
    </>
  )
}

Notification.propTypes = {
  error: PropTypes.string
}

export default Notification
