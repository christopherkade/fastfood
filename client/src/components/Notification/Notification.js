import React from "react"
import styled from "styled-components"

const StyledNotification = styled.div`
  position: fixed;
  bottom: 1rem;
  right: 1rem;
  background-color: #D52941;
  padding: 1rem;
  border-radius: 4px;
  color: white;
`

const Notification = ({ error }) => {
  return (
    <>
      {
        error ?
          <StyledNotification>
            {error}
          </StyledNotification > : null
      }
    </>
  )
}

export default Notification
