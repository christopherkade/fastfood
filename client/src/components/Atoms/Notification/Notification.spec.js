import React from 'react'
import { render, cleanup } from 'react-testing-library'

import Notification from "./Notification"

afterEach(cleanup);

describe("<Notification />", () => {
  describe("Snapshots", () => {
    it("should match snapshot", () => {
      const { container } = render(<Notification />);
      expect(container).toMatchSnapshot();
    });
  })
})
