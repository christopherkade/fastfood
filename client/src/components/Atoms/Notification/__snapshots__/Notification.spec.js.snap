// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Notification /> Snapshots should match snapshot 1`] = `<div />`;
